proxym3u8.php

```diff
@@ -41,7 +41,8 @@
 
     $url = substr($url, 0, strpos($url, 'OPT:'));
 
-    $requestHeaders = getallheaders();
+    //why it is here?
```

Comment:
rollback you changes in this file, there was a bug, but your changes are incorrect